<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditSiteWidget</class>
 <widget class="QDialog" name="EditSiteWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtUserName"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Site Name:</string>
       </property>
       <property name="buddy">
        <cstring>txtSiteName</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Login Name (optional):</string>
       </property>
       <property name="buddy">
        <cstring>txtUserName</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Categories:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Comment (optional):</string>
       </property>
       <property name="buddy">
        <cstring>txtComment</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtSiteName"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Password Type:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="buddy">
        <cstring>cmbPasswordType</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="txtComment"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Site Counter:&lt;br&gt;(increase to get a &lt;br&gt;new password)</string>
       </property>
       <property name="buddy">
        <cstring>spnSiteCounter</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="spnSiteCounter">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>99999</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QComboBox" name="cmbPasswordType"/>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Sample Password:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblSamplePassword">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Password is determined by the User Name, Master Password, Site Name, Type and Counter</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtSiteName</tabstop>
  <tabstop>txtUserName</tabstop>
  <tabstop>txtComment</tabstop>
  <tabstop>spnSiteCounter</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditSiteWidget</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditSiteWidget</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbPasswordType</sender>
   <signal>activated(int)</signal>
   <receiver>EditSiteWidget</receiver>
   <slot>passwordTypeChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>431</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtSiteName</sender>
   <signal>textEdited(QString)</signal>
   <receiver>EditSiteWidget</receiver>
   <slot>checkInputValidity()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>passwordTypeChanged(int)</slot>
  <slot>checkInputValidity()</slot>
 </slots>
</ui>
